overview.html

```diff
@@ -27,7 +27,7 @@ <h1 class="page-header">Docs Overview</h1>
 
           <h2>Protocol Specs and APIs</h2>
 
-          <p>To dive into the technical specs, see the <a target="_blank" href="https://github.com/interledger/rfcs">Interledger RFCs</a>. Also see the documentation for the components of the reference implementation, code-named <a target="_blank" href="https://github.com/interledger/five-bells-demo#why-five-bells">"Five Bells"</a>. There are the <a href="{{site.interledgerjs_baseurl}}js-ilp-connector/apidoc/">connector</a>, the example <a href="{{site.interledgerjs_baseurl}}five-bells-ledger/apidoc/">ledger</a> and the application layer <a href="{{site.interledgerjs_baseurl}}five-bells-wallet/apidoc/">wallet</a>.</p>
+          <p>To dive into the technical specs, see the <a target="_blank" href="https://github.com/interledger/rfcs">Interledger RFCs</a>. Also see the documentation for the components of the reference implementation, code-named <a target="_blank" href="https://github.com/interledger/five-bells-demo#why-five-bells">"Five Bells"</a>. There are the <a href="{{site.interledgerjs_baseurl}}js-ilp-connector/apidoc/">connector</a>, the example <a href="{{site.interledgerjs_baseurl}}five-bells-ledger/apidoc/">ledger</a> and the application layer <a href="{{site.interledgerjs_baseurl}}five-bells-wallet/apidoc/">ILP Kit</a>.</p>
```

Comment:
Do you know how to change the ILP Kit docs to compile to an `ilp-kit` directory instead of `five-bells-wallet`?